gtk/gtk_since_3_16.go

```diff
@@ -172,6 +172,23 @@ type ModelButton struct {
 	 return wrapModelButton(glib.Take(unsafe.Pointer(c))), nil
  }
 
+/*
+ * GtkPopover
+ */
+
+// FIXME: the following two funcs are deprecated since version 3.22.
+// But putting them there will break builds targettng GTK 3.14 and earlier.
```

Comment:
How come? Builds targeting 3.14 and earlier would use a tag gtk_3_14 or lower.
These would compile for anything less than 3.22 then.
Or am I missing something here?

Reply:
Oh, right, nevermind. I got that now. duh..
For anyone stumbling over this: See #516